controller/src/main/java/org/jboss/as/controller/management/Capabilities.java

```diff
@@ -40,4 +40,6 @@
 
     public static final String SSL_CONTEXT_CAPABILITY = "org.wildfly.security.ssl-context";
 
+    public static final String MANAGEMENT_SECURITY_REALM_CAPABILITY = "org.wildfly.core.management.security.realm";
```

Comment:
I think we'll want to change this name but as long as no one is going to use this outside of core before the core release next week, I don't care for now.

We also need wildfly-capabilities entries for all the elyron capabilities so it is easy to see how this name fits in with the others.